sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -95,9 +126,9 @@ options.Converters.Add(new ModelJsonConverter());
 DogListProperty dog = System.Text.Json.JsonSerializer.Deserialize<DogListProperty>(json, options);
 ```
 
-## Envelope BYOM Case
+## Envelope bring your own model case
```

Comment:
I've got no idea what this title means.   The core scenario here seems useful - how to use a converter to customize things.   We should consider revising the title so that it describes the scenario.

Reply:
I don't think `Envelope` or `BYOM` mean anything to customers.  I'd change this to just `Generic Models`.